<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyframes</class>
 <widget class="QWidget" name="Keyframes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MediaControl" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/dialogs/keyframe</normaloff>:/dialogs/keyframe</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Key frame</string>
       </property>
       <property name="class" stdset="0">
        <string>heading</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>238</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Key properties</string>
         </property>
         <property name="class" stdset="0">
          <string>section</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="Label" name="label">
           <property name="text">
            <string>Key Time</string>
           </property>
           <property name="class" stdset="0">
            <string>item</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Label" name="label_2">
           <property name="text">
            <string>Key Value</string>
           </property>
           <property name="class" stdset="0">
            <string>item</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="Label" name="label_3">
           <property name="text">
            <string>Interpolation</string>
           </property>
           <property name="class" stdset="0">
            <string>item</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="SpinBox" name="spinBox">
           <property name="suffix">
            <string>F</string>
           </property>
           <property name="maximum">
            <number>300000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="DoubleSpinBox" name="doubleSpinBox"/>
         </item>
         <item row="2" column="1">
          <widget class="ComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Spline</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Step</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="Label" name="label_4">
           <property name="text">
            <string>Lock Time</string>
           </property>
           <property name="class" stdset="0">
            <string>item</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Label" name="label_6">
           <property name="text">
            <string>Lock Value</string>
           </property>
           <property name="class" stdset="0">
            <string>item</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_2"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaControl</class>
   <extends>QPushButton</extends>
   <header>mediacontrol.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>doublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
